src/common/paths/delete.yml

```diff
@@ -47,15 +59,14 @@ post:
       If you don't pass a `predicate`, InfluxDB deletes all data with timestamps
       in the specified time range.
 
-      #### Related guides
```

Comment:
This is written on line 37. I removed it, but if we want to keep a `Related guides` section here, I can keep it in place.